models/apartment.rb

```diff
@@ -0,0 +1,3 @@
+class Apartment
+
+end
```

Comment:
missing an initialize method and any getters/setters you would want ie (`attr_accessors`)